packages/utils/cli/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@spark-ui/cli-utils",
```

Comment:
I am wondering if we should not dedicate this package to scaffolding other packages and rename it to something less generic ?

Reply:
I think it's nice to have something global like `spark [command]`, we should add each command in the package.json config. What do you think?

Reply:
What if there is another directory for tool packages called `tools` like `hooks` and `components`. Maybe not initially but if we consider to become an standard in the company this tool will be useful to create brand components following the same pattern

Reply:
Regarding directory (workspace) we can always update later it won't impact the package name that is published :) 

(What I mean is we can always change it later if it is merged as is)